### DISCUSSION TITLE
Update/Upgrade nzyme 1.1.1 to 1.2.2

### DISCUSSION THREAD

**Discussion by Falconcrest** [nzymedefense/nzyme#598] [Questions] [answered]
Hello,

How can I update / upgrade nzyme from version 1.1.1 to the latest version 1.2.2 without losing data?

Thanks

**Comment by lennartkoopmann** [comment, marked as answer]
Hi,

the blog posts have a short upgrading section all the way at the end. You can simply install the new `.deb` and no migration is required.

**Reply by ghost** [reply]
I found I had to reboot system and it came back online. I guess there's some services needing to be restarted.

**Reply by lennartkoopmann** [reply]
Yes, you have to restart nzyme after you install a new version: `systemctl restart nzyme`

**Comment by Falconcrest** [comment]
Sorry for my useless question!
Those who can read have a clear advantage!
I just found the solution in the excellent documentation.
Thank you for the great work!
